### ISSUE TITLE
Missing `boto3` dependency

### ISSUE THREAD

**Issue by Andrew-S-Rosen** [AgnostiqHQ/covalent#1867]
### Environment

- **Covalent version**: 0.230.0.rc0
- **Python version**: 3.8-3.10
- **Operating system**: Linux

### What is happening?

When running `covalent start`, I am getting

```bash
File "/opt/hostedtoolcache/Python/3.9.18/x64/bin/covalent", line 5, in <module>
    from covalent_dispatcher._cli.cli import cli
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/covalent_dispatcher/_cli/cli.py", line 27, in <module>
    from .groups import db, deploy
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/covalent_dispatcher/_cli/groups/__init__.py", line [17](https://github.com/Quantum-Accelerators/covalent-hpc-plugin/actions/runs/7010292329/job/19070495645?pr=57#step:5:18), in <module>
    from .deploy import deploy
  File "/opt/hostedtoolcache/Python/3.9.[18](https://github.com/Quantum-Accelerators/covalent-hpc-plugin/actions/runs/7010292329/job/19070495645?pr=57#step:5:19)/x64/lib/python3.9/site-packages/covalent_dispatcher/_cli/groups/deploy.py", line 25, in <module>
    import boto3
ModuleNotFoundError: No module named 'boto3'
```

### How can we reproduce the issue?

Run `pip install covalent==0.230.0.rc0` and then `covalent start`

### What should happen?

Covalent server should start.

### Any suggestions?

Add `boto3` to `requirements.txt` or modify the import statement [here](https://github.com/AgnostiqHQ/covalent/blob/fb360b4df9ce4bda531f5f4eac2978140c640f90/covalent_dispatcher/_cli/groups/deploy.py#L25).

**Comment by santoshkumarradha** [comment]
@kessler-frost I assume this was resolved? 

@Andrew-S-Rosen , if not this should be resolved soon :D

**Comment by Andrew-S-Rosen** [comment]
Looks like this was solved in #1866.
